collect_app/src/main/res/layout/form_entry_end.xml

```diff
@@ -17,116 +17,53 @@ the specific language governing permissions and limitations under the License.
-                android:padding="@dimen/margin">
-
-                <ImageView
-                    android:id="@+id/instance_name_info_icon"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:src="@drawable/ic_outline_info_24"
-                    app:layout_constraintStart_toStartOf="parent"
-                    app:layout_constraintTop_toTopOf="parent"
-                    app:tint="?colorPrimary" />
+            android:textStyle="bold"
+            app:layout_constraintBottom_toTopOf="@+id/save_as_draft"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintVertical_bias="0.5"
+            app:layout_constraintVertical_chainStyle="packed"
+            tools:text="You are at the end of All widgets" />
 
-                <com.google.android.material.textview.MaterialTextView
-                    android:id="@+id/instance_name_info_text"
-                    android:layout_width="0dp"
-                    android:layout_height="wrap_content"
-                    android:layout_marginStart="@dimen/margin"
-                    android:text="@string/manual_instance_name_warning"
-                    android:textAppearance="?textAppearanceBodyLarge"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintStart_toEndOf="@id/instance_name_info_icon"
-                    app:layout_constraintTop_toTopOf="@id/instance_name_info_icon" />
-
-                <com.google.android.material.button.MaterialButton
-                    android:id="@+id/instance_name_learn_more"
-                    style="@style/Widget.Material3.Button.TextButton"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:text="@string/manual_instance_learn_more_button"
-                    app:layout_constraintBottom_toBottomOf="parent"
-                    app:layout_constraintEnd_toEndOf="parent"
-                    app:layout_constraintTop_toBottomOf="@id/instance_name_info_text" />
-
-            </androidx.constraintlayout.widget.ConstraintLayout>
-        </com.google.android.material.card.MaterialCardView>
-
-        <TextView
-            android:layout_marginTop="@dimen/margin"
-            android:id="@+id/save_form_as"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:gravity="start"
-            android:lineSpacingMultiplier="1.2"
-            android:text="@string/save_form_as"
+        <com.google.android.material.button.MaterialButton
+            android:id="@+id/save_as_draft"
+            style="Widget.Material3.Button.OutlinedButton"
+            android:backgroundTint="@color/surfaceButtonColor"
```

Comment:
You shouldn't need the `backgroundTint` or `textColor` now that this is using the `OutlinedButton` style.

Reply:
Done.